### DISCUSSION TITLE
Translator donations

### DISCUSSION THREAD

**Discussion by BenLubar** [ReactiveDrop/reactivedrop_translations#905] [General]
Hi everyone! I'm setting up donations to translators, similar to what we have for campaign authors. There will also be a few other categories of donations added at the same time.

I need:

- What name you want to be called in the donation item
- An image I can use to make an icon to represent you (I can't use some of your Steam avatars because they're copyrighted characters)
- You to follow the steps at https://developer.reactivedrop.com/donation-items.html so I can mark the money from the item as going to you (you can send the link here or privately, it doesn't matter, only AS:RD admins can see it and there's no sensitive information on the page)

Examples of what the icons look like:

![translator_steffo](https://github.com/ReactiveDrop/reactivedrop_translations/assets/4257305/cdf4d8f4-5708-4e02-b43f-027bf960f961)
![translator_efi](https://github.com/ReactiveDrop/reactivedrop_translations/assets/4257305/8d422141-790b-4612-a7f5-8000003ed7ce)
![translator_trick-master](https://github.com/ReactiveDrop/reactivedrop_translations/assets/4257305/5d57ef97-8d0d-4454-90f9-0e4f1a672030)

Some of you are in countries with banking systems Steam can't talk to due to sanctions, and I wish I had an answer for how to proceed.

@geotavros @mithrand0 @Rayman1103 @Steffo99 @MakinDay @TheScientistPT @FurtadoPires @MartLegion @efimero @DukeDukeEm @NGCHunter2 @BRAIN-IS-BACK @Deerlord0 @zhongqingsong @ywgATustcbbs @wawawawawawawa @HagonWyvon (and also anyone I forgot)

**Comment by Steffo99** [comment]
What size and aspect ratio are the images in the stamp in?

**Reply by BenLubar** [reply]
I've got the template set up for 1:1 or 4:3, I resize to 408 wide and rotate 1.5 degrees so size doesn't matter. Also your avatar is already fine :P

**Reply by Duke-M-commits** [reply]
I personally would prefer an 1:1 poststamp frame for the avatars. As all the originals are square, you would have to find an good looking cropping for everyone of these.

**Comment by Deerlord0** [comment]
Umm, I need to make a business bank account and very own icon.
Fee for receiving money from Steam costs more than 10$ with personal account.

**Reply by Deerlord0** [reply]
I got AI art for myself.
![aideer2909611396](https://github.com/ReactiveDrop/reactivedrop_translations/assets/108241809/95adfb55-d7a3-40f0-a9b1-5d436b476704)

I have read the license. As noted on section III-6, it seems legit to earn money from generated image.
https://huggingface.co/lunarfish/furrydiffusion

**Reply by Deerlord0** [reply]
https://steamcommunity.com/sharedfiles/filedetails/?id=2979899568
My new bank account is ready. This is the workshop item I created by following instructions above.
I hope nothing wrong there.

**Reply by BenLubar** [reply]
All looks good!

**Comment by Duke-M-commits** [comment]
![image](https://github.com/ReactiveDrop/reactivedrop_translations/assets/69652463/3aaac088-ba7b-48c3-806f-73d99f5a67e2)

made with freeimage.ai

I named myself "Duke M" in my [latest PR](https://github.com/ReactiveDrop/reactivedrop_translations/pull/916) as this is my name on Steam and Github

**Comment by Duke-M-commits** [comment]
`"name_english": "Donor: Dmitriy"`

Sounds misleading in my eyes. It sounds like Dmitriy is the Donor.

How about `Donation to Dmitriy` or `Dmitriy Donor` or `Donation: Dmitriy`?

**Reply by NGCHunter2** [reply]
Adding to this, wouldn't it be beneficial to include the language the translator contributed to in the stamp title ([ITA], [JPN], [GER] etc.] so that potential donors wouldn't have to consult each individual stamp description?

**Reply by BenLubar** [reply]
You can search item descriptions on the store: https://store.steampowered.com/itemstore/563560/browse/?searchtext=italian&filter=all

**Comment by BRAIN-IS-BACK** [comment]
?
What on earth is this post all about?

**Reply by Duke-M-commits** [reply]
It's about the ability to donate Steam-money. To volunteer contributors.
@BenLubar are there any contributors which aren't volunteers?

**Reply by BenLubar** [reply]
Nope, everyone's a volunteer.

**Comment by FurtadoPires** [comment]
I personally don't want the money from donations, I did the Brazilian translation (PT-BR) by myself (and others who helped later) in a period where I was unemployed and I did for the sole reason of helping people who don't speak english to enjoy this game more.

I rather having any donation money from PT-BR translations to go to @BenLubar and the team who does the amazing job to keep this project alive, or use it to host official servers in the south america region.

Just keep my name in the credits and I'm good :)

**Comment by TheScientistPT** [comment]
Thank you very much, but I also don't want any donation money, I'm more than glad to just help translate the game in my spare time :)
